applications/app/controllers/LatestIndexController.scala

```diff
@@ -16,8 +16,11 @@ object LatestIndexController extends Controller with ExecutionContexts with impl
     loadLatest(path).map { _.map { index =>
       index.page match {
         case tag: Tag if tag.isSeries || tag.isBlog => index.trails.headOption.map(latest => {
-          if (request.isEmail) Found(latest.metadata.url + "/email")
-          else                 Found(latest.metadata.url)
+          if (request.isEmail) {
+            request.campaignCode.map(code => Redirect(latest.metadata.url + "/email", Map("CMP" -> Seq(code))))
```

Comment:
without the type it gives me this... :(
`type mismatch;
 found   : (String, Seq[String])
 required: (Nothing, Nothing)`
is there a way around that?